nightwatch.conf.js

```diff
@@ -0,0 +1,18 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const testSettings = require('./packages/terra-toolkit/lib/index').testSettings;
```

Comment:
We should probably stop using relative paths in the monorepo.

This code _should_ work the same:
```
const testSettings = require('terra-toolkit').testSettings;
```